### DISCUSSION TITLE
I want to help complete the friv library

### DISCUSSION THREAD

**Discussion by HardPowerhtml** [Tsyuay/FrivReborn#72] [Q&A]
Hi Tsyuay, I would also like to help you complete all the missing games in the friv library, I also worked with the friv library,

I have a experience with html and swf games

I want to support the project so that it is completed

and I hope you accept my help

**Comment by Tsyuay** [comment]
Hi, I had a problem with a troll creating multiple accounts just to make game requests and non-sense pull requests, and I also don't want trolls in the project who want to request to contribute the project just to delete everything. So please tell me if you aren't one of them.

**Comment by HardPowerhtml** [comment]
rest assured, I'm not one of them, you can trust me! do not worry.

**Comment by Tsyuay** [comment]
done go to https://github.com/Tsyuay/FrivReborn/invitations if you don't get the notification of the invite

**Comment by HardPowerhtml** [comment]
thank you, I'll start soon!
